docs/src/pages/[platform]/components/grid/index.page.mdx

```diff
@@ -3,7 +3,6 @@ title: Grid
 metaTitle: Grid
 description: Grid provides a CSS grid container.
 metaDescription: Grid provides a CSS grid container.
-ariaPattern: https://www.w3.org/TR/wai-aria-practices-1.2/#grid
```

Comment:
Our grid component is presentational, not an ARIA grid (which has expectations of directional navigation, focusable content in each "cell", etc) so I removed it.

Reply:
Good catch. Do you think we should roadmap this to bring the component more inline with the ARIA spec?

Reply:
Hmm no, maybe as a separate, new component (like for creating an [editable spreadsheet](https://www.w3.org/WAI/ARIA/apg/example-index/grid/dataGrids.html#ex2_label) maybe?) but different from this purely CSS layout component.